util/cosmos/iscn/sign.js

```diff
@@ -215,7 +214,7 @@ export async function estimateISCNTxGas(tx) {
   const gasUsedEstimation = byteSize.multipliedBy(GAS_ESTIMATOR_SLOP).plus(interceptWithBuffer);
   return {
     gasFee: {
-      amount: [{ amount: gasUsedEstimation.toFixed(0, 0), denom: 'nanolike' }],
+      amount: [{ amount: gasUsedEstimation.multipliedBy(DEFAULT_GAS_PRICE_NUMBER).toFixed(0, 0), denom: 'nanolike' }],
       gas: gasUsedEstimation.toFixed(0, 0),
```

Comment:
is nanolike a const defined as COSMOS_DENOM?

Reply:
true! will change this.